### ISSUE TITLE
Laravel 5.5 artisan migrate:(dis)organise failure

### ISSUE THREAD

**Issue by Alarich** [JayBizzle/Laravel-Migrations-Organiser#20]
Greetings,

Thank you for bringing this to Laravel 5.5.
There appears to be an issue with the commands migrate:organise, migrate:disorganise.

Error:

```
[ReflectionException]
Method Jaybizzle\MigrationsOrganiser\Commands\MigrateOrganise::handle() does not exist
```

Instead of handle(), a method fire() exists. In the documentation for 5.0, it does indeed say the correct method is fire(), but it seems from 5.1 on, this has changed.

Link to source on the change from 5.1 - [https://mattstauffer.com/blog/creating-artisan-commands-with-the-new-simpler-syntax-in-laravel-5.1/](https://mattstauffer.com/blog/creating-artisan-commands-with-the-new-simpler-syntax-in-laravel-5.1/)

Everything else seems to work perfectly, thank you.

Sincerely,
Alarich

**Comment by JayBizzle** [comment]
Thanks for reporting, i will take a look into this when i get a minute in the next couple of days 👍

**Comment by JayBizzle** [comment]
Fixed in #21 and available in [v4.1.1](https://github.com/JayBizzle/Laravel-Migrations-Organiser/releases/tag/v4.1.1)

Thanks!
